docs/benchmarks/micro-benchmarks.md

```diff
@@ -0,0 +1,113 @@
+---
+id: micro-benchmarks
+---
+
+# Micro Benchmarks
+
+## Benchmarking list
+
+### Computation benchmark
+
+### Communication benchmark
+
+### Computation-communication benchmark
+
+### Storage benchmark
+
+
+## Benchmarking metrics
+
+<table>
+  <tbody>
+    <tr align="center" valign="bottom">
+      <td>
+      </td>
+      <td>
+        <b>Micro Benchmark</b>
+        <img src={require('../assets/bar.png').default}/>
+      </td>
+      <td>
+        <b>Model Benchmark</b>
```

Comment:
why don't remove 'Model Benchmark'?

Reply:
it's placeholder, to be updated by tingting